### PR TITLE
Remove ip address blocking from application

### PR DESCRIPTION
**Pull request by JoeCohen**
- To be handled by Ubuntu Iptables instead of Rails
- (There are currently no addresses blocked in Rails.)
- See 1st two posts in [MO Developers › blocking DOS attacks in the application](https://groups.google.com/forum/#!topic/mo-developers/icP-cDw3eSs)
- Delivers [Pivotal #111721820](https://www.pivotaltracker.com/story/show/111721820)

### CODE REVIEW

**Comment by JoeCohen** [comment]
Arg.  Another Travis-only failure.  I will look at this and see if I can fix.  
It passes locally on the VM:
```
VM:mushroom-observer$ git branch
...
* remove-ip-blocking
...
VM:mushroom-observer$ rake
...Started with run options --seed 8977
...
  1220/1220: [=============================================] 100% Time: 00:02:33, Time: 00:02:33
...
1220 tests, 17202 assertions, 0 failures, 0 errors, 0 skips
```

**Comment by JoeCohen** [comment]
TL;DR:  merge if you agree with the approach of blocking IP addresses in the Iptables, rather than the application.  See 1st two posts in [MO Developers › blocking DOS attacks in the application](https://groups.google.com/forum/#!topic/mo-developers/icP-cDw3eSs)
Long note re intermittent?, order-dependent?, Travis-only failure:
This now passes. But:
- Before I made the PR, this branch passed locally;
- When I pushed it to my GitHub repo, the Travis job passed.  See [JoeCohen Travis Build128](https://travis-ci.org/JoeCohen/mushroom-observer/builds/102456392), but
- When I first made the PR, the Travis build Errored in the MO repo.  See[MO Travis Build 689](https://travis-ci.org/MushroomObserver/mushroom-observer/builds/102456491).
I have no idea of the ultimate cause of that Error, I have no idea why it's now behaving itself, and I have no idea how to debug something which occurs only in a build on the MO repo.
I don't think the (original Error)[https://travis-ci.org/MushroomObserver/mushroom-observer/builds/102456491] or the current passing state have anything to do with my code. I think it's either a one-time glitch or a combination: order dependency, Travis, and the particular states of the code and the db.
